Fix compilation of 3.16, picked from
https://github.com/axboe/fio/commit/5b215853ed4b438b5b2d4ac3e56d5f0d19e145d9
Can be removed once 3.17+ is released.
Index: fio/gclient.c
===================================================================
--- fio.orig/gclient.c
+++ fio/gclient.c
@@ -330,7 +330,7 @@ static void gfio_update_thread_status_al
 	static char message[100];
 	const char *m = message;
 
-	strncpy(message, sizeof(message), "%s", status_message);
+	snprintf(message, sizeof(message), "%s", status_message);
 	gtk_progress_bar_set_text(GTK_PROGRESS_BAR(ui->thread_status_pb), m);
 	gtk_progress_bar_set_fraction(GTK_PROGRESS_BAR(ui->thread_status_pb), perc / 100.0);
 	gtk_widget_queue_draw(ui->window);
